app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/PlaybackView.kt

```diff
@@ -197,7 +202,10 @@ class PlaybackView(val state: RHMIState, val controller: MusicController, val ca
 			state.getArtistImageModel()?.asRaImageModel()?.value = carAppImages["${musicImageIDs.ARTIST}.png"]
 			state.getAlbumImageModel()?.asRaImageModel()?.value = carAppImages["${musicImageIDs.ALBUM}.png"]
 
-			buttons[3].setVisible(false)
+			// setting the actions button icon since the button has a book icon by default
+			customActionButton.getImageModel()?.asImageIdModel()?.imageId = musicImageIDs.ACTIONS
+
+			repeatButton!!.getAction()?.asRAAction()?.rhmiActionCallback = RHMIActionCallback { controller.toggleRepeat() }
```

Comment:
Please use the null-coalescing `repeatButton?.getAction` style, just in case something accidentally doesn't get initialized earlier.

Reply:
Ah not sure why I did !! there instead of ?. , changed that